src/manifest.js

```diff
@@ -43,7 +44,35 @@ export const parseManifest = ({
   parser.push(manifestString);
   parser.end();
 
-  return parser.manifest;
+  const manifest = parser.manifest;
+
+  // remove llhls features from the parsed manifest
```

Comment:
This is so that we can turn off llhls by default, so that we can merge this and continue to play llhls back how we do now with just segments. Then we can remove this option when llhls is on by default.